classification1.Rmd

```diff
@@ -472,86 +489,155 @@ perim_concav + annotate("path",
 ### More than two explanatory variables 
 
 Although the above description is directed toward two predictor variables, 
-exactly the same K-nearest neighbours algorithm applies when you
+exactly the same $K$-nearest neighbors algorithm applies when you
 have a higher number of predictor variables (i.e., a higher-dimensional
```

Comment:
(i.e. a higher-dimensional predictor space)

do we ever use the word "dimension" or "space" anywhere else in the book? If not, should remove it in this little blurb here.

If so, should consider whether there is simpler language to use here.